app/models/ebean/Alert.java

```diff
@@ -0,0 +1,201 @@
+/**
+ * Copyright 2015 Groupon.com
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package models.ebean;
+
+import com.avaje.ebean.Model;
+import com.avaje.ebean.annotation.CreatedTimestamp;
+import com.avaje.ebean.annotation.UpdatedTimestamp;
+
+import java.sql.Timestamp;
+import java.util.UUID;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import javax.persistence.Version;
+
+/**
+ * Data model for alerts.
+ *
+ * @author Deepika Misra (deepika at groupon dot com)
+ */
+@Entity
+@Table(name = "alerts", schema = "portal")
+public class Alert extends Model {
+
+    @Id
```

Comment:
I think you are considering this too much in a file to db context. You should think of this as an independent alert repository implementation. When inserting a new alert the UUID needs to be generated somewhere. Brandon is saying it can be done at the database level.

However, taking a step back I still want to ensure we understand the use of UUID as PK versus a raw integer as the PK and an external uuid as a secondary id. The internal id as an identifier for the database it is going to be used in joins (for example). Although possible using a UUID for joins is not ideal in my opinion. 

@BrandonArp what are your thoughts here on the above?